ruby_on_rails/send_emails.md

```diff
@@ -33,19 +21,68 @@ class ApplicationMailer < ActionMailer::Base
 end
 ```
 
-* add the following to `config/application.rb`
+- add the following to `config/application.rb`
 
 ```ruby
 config.action_mailer.default_url_options = { host: ENV['APP_HOST'], port: ENV['APP_PORT'] }
 ```
 
-* add the following to `config/environments/development.rb`:
+- add the following to `config/environments/development.rb`:
 
 ```ruby
 config.action_mailer.delivery_method = :letter_opener
 ```
 
-* add the following `config/environments/production.rb`:
+Now either setup SendGrid or SparkPost. If you need to send more than 100 mails per day use SparkPost otherwise use SendGrid. (It's more reliable and easier to set up.) (Sparkpost sometimes fails to send mails to outlook and bluewin email addresses.)
```

Comment:
mmmm. why did we use sendgrid the first time? should we just switch to it? I don't see sendgrid much more difficult to configure.

Reply:
I think this should be discussed in a wg-operations meeting. can you please add this point to the protocol, please?